psh/echo/echo.c

```diff
@@ -59,7 +59,6 @@ static int psh_echo(int argc, char **argv)
 {
 	int c, i, argend = argc;
 	size_t j;
-	FILE *output = stdout;
 
 	while ((c = getopt(argc, argv, "h")) != -1) {
```

Comment:
I think we can apply **MISRA** here.

Reply:
Hmm, the loop not touched, it's out of scope of the PR.